compiler/rustc_span/src/symbol.rs

```diff
@@ -158,6 +158,7 @@ symbols! {
     // There is currently no checking that all symbols are used; that would be
```

Comment:
The above paragraph is now out-of-date:
- The proc macro will only abort if symbols are duplicated (I assume the duplication detection still works?)
- The Vim instructions are no longer correct and should just be removed.

Reply:
Did something go wrong? It looks like the paragraph is still there.

Reply:
Oh woops 🫢
Sorry about that.
I think I haven't completely woken up yet today 😅